sync/class.jetpack-sync-settings.php

```diff
@@ -53,12 +54,28 @@ static function get_setting( $setting ) {
 			return self::$settings_cache[ $setting ];
 		}
 
-		$value = get_option( self::SETTINGS_OPTION_PREFIX . $setting );
+		if ( self::is_network_setting( $setting ) ) {
+			if ( is_multisite() ) {
+				$value = get_site_option( self::SETTINGS_OPTION_PREFIX . $setting );
+			} else {
```

Comment:
Is it a real use case to sync a network setting for a single site?

Reply:
These settings determine are there to configure how sync works. 
I want single sites to not change those settings and always rely on the default configuration. 

The main idea is to simplify things.

Reply:
Thanks for clarifying 👍